src/python/pants/backend/python/tasks/pytest_prep.py

```diff
@@ -56,10 +66,7 @@ def extra_requirements(self):
 
   def extra_files(self):
     yield self.ExtraFile.empty('pytest.ini')
-
-    enclosing_dir = os.path.dirname(__name__.replace('.', os.sep))
-    plugin_path = os.path.join(enclosing_dir, 'coverage/plugin.py')
-    yield self.ExtraFile(path=plugin_path,
+    yield self.ExtraFile(path='{}.py'.format(self.PytestBinary.coverage_plugin_module()),
```

Comment:
This is going to refer to a different file than it used to... but the tests are passing, so I'm going to assume that that is intentional?

Reply:
The tests ride above the layer of the task, they just create it and execute it and so plugin details (for both py.test and coverage) are opaque - though important - to the test. The key paired change to this one is in `PytestRun` where `PytestPrep.PytestBinary.coverage_plugin_module()` is used to setup the `coverage.rc` ini file.